corehq/apps/case_importer/management/commands/update_case_property.py

```diff
@@ -13,11 +13,12 @@
 class Command(BaseCommand):
 
     def add_arguments(self, parser):
-        parser.add_arugment('infile')
-        parser.add_arugment('logfile')
```

Comment:
did this just never work?

Reply:
I had made a fix in another branch and deployed a private release with that when I ran the command for ICDS. That was the night I left for vacation, and I had forgotten to PR it when I got back.